src/sandbox/wiktor/HistoryTable.tsx

```diff
@@ -0,0 +1,106 @@
+import CallMadeIcon from '@mui/icons-material/CallMade';
+import CallReceivedIcon from '@mui/icons-material/CallReceived';
+import Chip from '@mui/material/Chip/Chip';
+import Paper from '@mui/material/Paper';
+import { DataGrid, GridColDef, GridRenderCellParams } from '@mui/x-data-grid';
+import React from 'react';
+import styled from 'styled-components'; // useTheme
+
+import { HistoryTableData } from './sampleHistoryTableData';
+
+const Container = styled.div`
+  max-width: 800px;
+  padding: 1rem;
+  margin: 1rem;
+`;
+
+type TableProps = {
+  data: HistoryTableData[];
+};
+
+const Table: React.FC<TableProps> = (props) => {
+  //   const classes = useStyles();
+
+  const columns: GridColDef[] = [
```

Comment:
Przenieś to do pliku `columns.tsx` jako zwracana wartość z `getTableColumns`

```suggestion
  const columns = getTableColumns()
```

Reply:
corrected